pkg/api/api.go

```diff
@@ -250,15 +250,15 @@ func (hs *HTTPServer) registerRoutes() {
 		// current org without requirement of user to be org admin
 		apiRoute.Group("/org", func(orgRoute routing.RouteRegister) {
 			lookupEvaluator := func() ac.Evaluator {
-				if hs.Cfg.IsEnterprise {
+				if hs.License.FeatureEnabled("accesscontrol.enforcement") {
```

Comment:
nit: could you use the `FeatureAccessControlEnforcement` constant instead of typing out the string? Same goes for the other checks.

Reply:
No that one is only in enterprise code. I think we should move them but in another pr?

Reply:
Ah, gotcha, that makes sense then. Always annoying to move constants etc across from enterprise to OSS. Same for tests - I was about to suggest that you use the existing `ValidLicense` test struct, but it lives in enterprise and not OSS 😢